target/linux/ramips/dts/mt7620a_hiwifi_hc5661.dts

```diff
@@ -1,34 +1,43 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
```

Comment:
Please do not add a copyright header to a file which was also edited by others. For new ones we should add it.

Reply:
Fixed. Thanks!